src/type_definitions.jl

```diff
@@ -145,16 +145,17 @@ function GenericSpace(conversion::Function; kwargs...)
     mins = Any[]
     maxs = Any[]
     bounded = Bool[]
-    j = 1
-    for (key, value) in kwargs
+
+    for (j, (key, value)) in enumerate(kwargs)
         nameindices[key] = j
         push!(dims, conversion(value))
         push!(indices, 1:length(value))
         push!(mins, minimum(value))
         push!(maxs, maximum(value))
-        push!(bounded, false)
-        j += 1
+        # always true, we are considering only discretized states
+        push!(bounded, true)
```

Comment:
For now, since states are discrete, bounded should be always true to check if we are going in a unfeasible state (and out-of-bound of the interpolation)